docs/experiments-car18.md

```diff
@@ -0,0 +1,37 @@
+# Anserini Experiments on TREC Core
```

Comment:
Change this

Reply:
Also, do not forget to update the names in the doc such as `CarCollection` after you finish everything.